Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -244,7 +364,7 @@ def options_handler():
 def main():
     install_logging('cleanup_cloud_instance.log', logger=logging)
 
-    # in cloud we dont use demisto username
+    # In Cloud, We don't use demisto username
```

Comment:
```suggestion
    # In Cloud, we don't use demisto username
```

Reply:
my grammar is correct :)
https://www.grammarly.com/blog/capitalization-after-colons